src/helpers/makeQuery.js

```diff
@@ -47,7 +52,12 @@ const makeQuery = <P: mixed, M: mixed>(
     );
 
     if (results.length === 0) {
-      throw new ErrorWithStack('No instances found', getAllBy);
+      throw new ErrorWithStack(
+        `No instances found with ${name} '${
+          typeof matcher === 'string' ? matcher : JSON.stringify(matcher) || ''
```

Comment:
we have `pretty-format` in our deps, so let's use that with `{min: true}` option set: https://github.com/facebook/jest/tree/master/packages/pretty-format#usage-with-options